packages/mobile/src/screens/favorites-screen/DownloadProgress.tsx

```diff
@@ -48,17 +44,8 @@ export const DownloadProgress = () => {
     (status) => status === OfflineDownloadStatus.SUCCESS
   ).length
 
-  const isMarkedForDownload = useSelector(
-    getIsCollectionMarkedForDownload(DOWNLOAD_REASON_FAVORITES)
-  )
-
   // Only render if there are active downloads
-  if (
-    numDownloadsComplete === numDownloads ||
-    !isMarkedForDownload ||
-    !isReachable
-  )
-    return null
+  if (numDownloadsComplete === numDownloads || !isReachable) return null
```

Comment:
I suppose i made this change here

https://github.com/AudiusProject/audius-client/pull/2762

Just want to make sure with the redux persist changes, we've set this up in a way where this will not show until you have marked stuff for download (ie that numDownloads & numDownloadsComplete are both 0 until flipping the switch)